docs/resources/dashboard.md

```diff
@@ -112,6 +112,7 @@ The `chart` mapping supports the following:
 * `summarization` - (Required) Summarization strategy for the chart. MEAN is default. Valid options are, `MEAN`, 
 `MEDIAN`, `MIN`, `MAX`, `SUM`, `COUNT`, `LAST`, `FIRST`
 * `description` - (Optional) Description of the chart
+* `base` - (Optional) The base of logarithmic scale charts. Usually 10. Omit or set to 0 for linear scale.
```

Comment:
Might want to clarify `Usually 10.`. That sounds like a default but then you go on to say `Omit or 0 for linear scale`. Perhaps saying `Omit or set to 0 for the default linear scale. Usually set to 10 for traditional logarithmic scale.`.

Reply:
Thank you for the feedback. I changed the wording in the doc as you suggested.